packages/dts-generator/src/js-utils/ui5-metadata.js

```diff
@@ -148,11 +207,11 @@ async function downloadDtsgenrcFiles(libs, version, outDir, prepareDir = false)
  * @returns {Promise<any>}
  */
 async function loadJSON(file) {
-  let content = await readFile(file, "utf8");
+  let content = await fs.readFile(file, "utf8");
   if (file.endsWith(".dtsgenrc")) {
-      const stripJsonComments = (await import("strip-json-comments")).default;
-      // allow comments in .dtsgenrc files
-      content = stripJsonComments(String(content));
+    const stripJsonComments = (await import("strip-json-comments")).default;
```

Comment:
There`s also a parser for JSON with comments: https://www.npmjs.com/package/comment-json

Reply:
Ok, but that's the choice between using one or the other third-party package and in this case I prefer the one from sindresorhus with 50M weekly downloads and newer latest release over the other one with 2M weekly downloads.